prisma/schema.prisma

```diff
@@ -1649,3 +1650,29 @@ model SNSIntegration {
 model SecurityCheckIdent {
   id String @id @unique @default(uuid()) @db.Uuid
 }
+
+// File represents a proxied file in the database.
+model File {
+  // base fields
+  id        String    @id @unique @default(uuid()) @db.Uuid
+  createdAt DateTime  @default(now())
+  updatedAt DateTime  @default(now()) @updatedAt
+  deletedAt DateTime?
+
+  // the name of the file
+  fileName String
+
+  // the relative path or URL pointing to the file
+  filePath String
```

Comment:
possible uniqueness constraint needed? tenant-file

Reply:
yeah, though functionally since we're going to be the main consumer of this API, we should be using UUIDs as filenames so there should be functionally no collisions but I see no harm in double checking that here.